### PR TITLE
Fix train_text_to_image_qat.py

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

- Fix incorrect format of `openvino/stable-diffusion/requirements.txt` 
- Change argument name of `Accelerator` from `logging_dir` to `project_dir`, `logging_dir` deprecated since 0.18
- Remove second call `accelerator.prepare` for `unet`

### CODE REVIEW

**Comment by AlexKoff88** [comment]
As far as I understood model export gets broken after this change so it needs to be fixed before merging this.

File: examples/openvino/stable-diffusion/requirements.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ accelerate
 diffusers
 torch~=1.13
```
Comment (fxmarty) [inline examples/openvino/stable-diffusion/requirements.txt:3]: Is there anything blocking for pytorch 2.0?

**Comment by AlexKoff88** [comment]
> Thank you!
> 
> I can't test those changes as I get an error at this point
> 
> https://github.com/huggingface/optimum-intel/blob/0a80e2053b26d72869710c5f2c14b9ee06d5141e/examples/openvino/stable-diffusion/train_text_to_image_qat.py#L648
> 
> in the script, telling me that the `collate_fn` from the dataloader is not pickable. Have you had the issue @AlexKoff88 ?

I am looking into it. There are multiple issues introduced by updates in `transformers` and `accelerate` libraries. Need to highly refactor the script.

**Comment by AlexKoff88** [comment]
Some issues in this PR should be fixed after merging this PR: https://github.com/huggingface/optimum-intel/pull/397

File: examples/openvino/stable-diffusion/requirements.txt (hunk 1 above)
Comment (AlexanderDokuchaev) [reply examples/openvino/stable-diffusion/requirements.txt:3]: Did not check it.

**Pull request closed without merging**